notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<p>* "You can create a custom handler by having a class with any name, but it must have an <code>initialize()</code> and a <code>handle()</code> method. " --&gt; "A handler is any class that has both an <code>initialize()</code> and a <code>handle()</code> method." </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
<p>Modified. Thanks!</p>